readme.txt

```diff
@@ -151,88 +148,148 @@ To change the form_title in this case, you need to add the following attribute t
+[Details](https://github.com/auth0/wp-auth0/blob/master/CHANGELOG.md#350-2018-01-25)
 
 = 3.4.0 =
 
-**Closed issues**
-- Remove connections interface [Github Issue 338](https://github.com/auth0/wp-auth0/issues/338)
-- Outdated php-jwt does not conform to specs, breaks my login flow [Github Issue 306](https://github.com/auth0/wp-auth0/issues/306)
-
-**Added**
-- Added Lock 11 / Auth0 9.0, Updated SSO, JWT Algorithm Upgrade Fixes [Github PR 350](https://github.com/auth0/wp-auth0/pull/350) - [cocojoe](https://github.com/cocojoe)
-- Add RS256 support [Github PR 331](https://github.com/auth0/wp-auth0/pull/331) - [renrizzolo](https://github.com/renrizzolo)
-
-**Fixed**
-- Switching wizard admin user creation to use /dbconnections/signup  [Github PR 356](https://github.com/auth0/wp-auth0/pull/356) - [joshcanhelp](https://github.com/joshcanhelp)
+[Details](https://github.com/auth0/wp-auth0/blob/master/CHANGELOG.md#340-2018-01-08)
 
 = 3.3.2 =
 
-**Added**
-- Added translation support for a few user-facing exception messages [Github PR 312](https://github.com/auth0/wp-auth0/pull/312) - [idpaterson](https://github.com/idpaterson)
-
-**Changed**
-- Use literal 'wp-auth0' rather than WPA0_LANG constant [Github PR 311](https://github.com/auth0/wp-auth0/pull/311) - [idpaterson](https://github.com/idpaterson)
-
-**Fixed**
-- Prevent script errors throughout the admin panel when anonymous data metrics are disabled [Github PR 328](https://github.com/auth0/wp-auth0/pull/328) - [idpaterson](https://github.com/idpaterson)
+[Details](https://github.com/auth0/wp-auth0/blob/master/CHANGELOG.md#332-2017-10-05)
 
 = 3.2.24 =
 
-**Changed**
-- More generous JWT leeway [Github PR 332](https://github.com/auth0/wp-auth0/pull/332) - [cocojoe](https://github.com/cocojoe)
-
-**Removed**
-- Remove client_id/secret validation since it is not allowed anymore [Github PR 334](https://github.com/auth0/wp-auth0/pull/334) - [glena](https://github.com/glena)
+[Details](https://github.com/auth0/wp-auth0/blob/master/CHANGELOG.md#3224-2017-08-14)
 
 = 3.2.23 =
 
-**Changed**
-- Update /authorize URL [Github PR 326](https://github.com/auth0/wp-auth0/pull/326) - [cocojoe](https://github.com/cocojoe)
+[Details](https://github.com/auth0/wp-auth0/blob/master/CHANGELOG.md#3223-2017-07-18)
 
 = 3.2.22 =
 
-**Fixed**
-- Fixed migration for older plugins that use base64 secret [Github PR 324](https://github.com/auth0/wp-auth0/pull/324) - [cocojoe](https://github.com/cocojoe)
+[Details](https://github.com/auth0/wp-auth0/blob/master/CHANGELOG.md#3222-2017-06-26)
 
 = 3.2.21 =
 
-**Added**
-- Improve redirect_login error logging, JWT leeway [Github PR 317](https://github.com/auth0/wp-auth0/pull/317) - [cocojoe](https://github.com/cocojoe)
+[Details](https://github.com/auth0/wp-auth0/blob/master/CHANGELOG.md#3221-2017-06-14)
+
+= 3.2.5 =
```

Comment:
It's nice to have a complete log. However, I'm not sure it adds any value going back before 3.0. Typically you only care about latest and greatest or at least back a few minors when you upgrade.
It can link off to changelog? for historical <3.x ?

Reply:
@cocojoe - removed